proto/ord_schema.proto

```diff
@@ -525,30 +516,30 @@ message ReactionOutcome {
   // Reaction time (for flow, equivalent to residence time or spacetime).
   Time reaction_time = 1;  
   // Conversion with respect to the limiting reactant.
-  float conversion_in_percent = 2;
+  Percentage conversion = 2;
   repeated ReactionProduct products = 3;
   repeated ReactionAnalysis analyses = 4;
 }
 
 message ReactionProduct {
   Compound compound = 1;
   bool is_desired_product = 2;
-  float yield_in_percent = 3;
-  float yield_precision_in_percent = 4;
-  ReactionAnalysis.AnalysisType yield_type = 5;
+  Percentage yield = 3;
+  ReactionAnalysis.AnalysisType yield_type = 4;
   // Used to describe an UNKNOWN yield type.
-  string yield_type_custom = 6;
-  float purity_in_percent = 7;
+  string yield_type_custom = 5;
+  float purity_in_percent = 6;
   enum SelectivityType {
     SELECTIVITY_TYPE_UNKNOWN = 0;
     EE = 1;  // Enantiomeric excess as a percentage.
     ER = 2;  // Enantiomeric ratio. (x:1)
     DE = 3;  // Diasteromeric ratio (x:1)
   }
-  SelectivityType selectivity_type = 8;
-  string selectivity_type_custom = 9;
-  float selectivity_value = 10;
-  float selectivity_precision_in_percent = 11;
+  SelectivityType selectivity_type = 7;
+  string selectivity_type_custom = 8;
+  // TODO(kearnes): Should these fields be replaced with Percentage?
```

Comment:
Only in the case of EE will the selectivity be a percentage. If we want to allow ER and DE to be specified (which I think we do), we'll have to leave it as a float. ```selectivity_precision_in_percent``` is a mouthful though

Reply:
OK, that's what I thought.

Should this just be `selectivity_precision` and maintain the "in the same units" paradigm? Although I'm not entirely sure what that means for a ratio...

Reply:
I don't know that I have ever seen EEs, DEs, ERs, etc. reported with uncertainty values. This would be a great question for Abby. I think you can safely change it to ```selectivity_precision``` and we can worry about what that means later

Reply:
Done.